Lombiq.Hosting.MediaTheme.Deployer/Constants/PathConstants.cs

```diff
@@ -8,12 +9,16 @@ public static class PathConstants
     public const string MediaThemeTemplatesDirectory = "Templates";
     public const string MediaThemeAssetsDirectory = "Assets";
 
+    // These need to use forward slashes on every platform.
     public const string MediaThemeAssetsWebPath = MediaThemeRootDirectory + "/" + MediaThemeAssetsDirectory;
-    public const string MediaThemeAssetsCopyDirectoryPath = "\\" + MediaThemeRootDirectory + "\\" + MediaThemeAssetsDirectory;
-
     public const string MediaThemeTemplatesWebPath = MediaThemeRootDirectory + "/" + MediaThemeTemplatesDirectory;
-    public const string MediaThemeTemplatesCopyDirectoryPath = "\\" + MediaThemeRootDirectory + "\\" + MediaThemeTemplatesDirectory;
```

Comment:
Well, _I_ don't use them, since that's pre-existing code :D. But yeah, that path is wrong. And yes, `Path.Combine()` is always preferable (`Path.Join()` is also pre-existing code), though here we are working with local file paths as well as path separators used by Orchard during import (which is forward slash), so we can't use `Path` everywhere.